tenancy_application_context.md

```diff
@@ -61,4 +63,83 @@ data | hash |
     }
   }
 }
-```
\ No newline at end of file
+```
+
+### TenancyApplication.Dossier.Received
+
+Field | Type | Content / Remarks
+---|---|---
+eventType | string | TenancyApplication.Dossier.Received
+data | hash |
+&nbsp;&nbsp;dossierId | string | Unique identifier within the tenancy application platform |
+&nbsp;&nbsp;unitReference | string | String referencing an existing unit in the target GARAIO REM |
+&nbsp;&nbsp;title | hash | containing the title of the application in 4 languages
+&nbsp;&nbsp;&nbsp;&nbsp;de | string | String containing the title of the application in german
+&nbsp;&nbsp;&nbsp;&nbsp;fr | string | String containing the title of the application in french
+&nbsp;&nbsp;&nbsp;&nbsp;it | string | String containing the title of the application in italian
+&nbsp;&nbsp;&nbsp;&nbsp;en | string | String containing the title of the application in english
+&nbsp;&nbsp;text | hash | containing the text of the application in 4 languages
+&nbsp;&nbsp;&nbsp;&nbsp;de | string | Html string containing the text of the application in german
+&nbsp;&nbsp;&nbsp;&nbsp;fr | string | Html string containing the text of the application in french
+&nbsp;&nbsp;&nbsp;&nbsp;it | string | Html string containing the text of the application in italian
+&nbsp;&nbsp;&nbsp;&nbsp;en | string | Html string containing the text of the application in english
+&nbsp;&nbsp;url | hash | containing the url of the application for 4 languages
```

Comment:
Brauchen wir die Url in 4 Sprachen? Aroov sollte das aufgrund des Spracheinstellung des angemeldeten Users selber steuern